packages/swagger-client/src/generators/__snapshots__/OperationGenerator.test.ts.snap

```diff
@@ -0,0 +1,42 @@
+// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html
+
+exports[`OperationGenerator > [GET] should generate code based on a pathParamsType \`inline\` 1`] = `
+"/**
+ * @summary Info for a specific pet
+ * @link /pets/:pet_id
+ */
+export async function showPetById<TData = ShowPetByIdQueryResponse>(
+  petId: ShowPetByIdPathParams['pet_id'],
+  testId: ShowPetByIdPathParams['testId'],
+  options: Partial<Parameters<typeof client>[0]> = {},
+): Promise<ResponseConfig<TData>> {
+  const { data: resData } = await client<TData>({
+    method: 'get',
+    url: \`/pets/\${petId}\`,
+    ...options,
+  })
+
+  return resData
+}
+"
+`;
+
+exports[`OperationGenerator > [GET] should generate code based on a pathParamsType \`object\` 1`] = `
+"/**
+ * @summary Info for a specific pet
+ * @link /pets/:pet_id
+ */
+export async function showPetById<TData = ShowPetByIdQueryResponse>(
+  { petId, testId }: ShowPetByIdPathParams,
```

Comment:
i kinda feel that the param names should be preserved verbatim from the schema in the object case. otherwise you have to understand not just that the names are converted to camelCase, but also how it handles the conversion for weird cases like e.g. `a_b_c`, `A_B_CD`, `_A__B_C` etc. the casing may have semantic meaning in the context of an API and this gets lost in the conversion.

Reply:
do query params get converted to camelCase?